packages/flutter_tools/templates/app_shared/.gitignore.tmpl

```diff
@@ -8,6 +8,7 @@
 .buildlog/
 .history
 .svn/
+migrate_working_dir/
```

Comment:
does it need to be in the app's directory? could it use the temp directory instead?

Reply:
The intention of this working dir is to be easily accessible and obvious. Putting it in a temp or hidden away folder makes it hard for developers to enter and work with the files inside.

Reply:
sgtm

Reply:
I specifically don't want this directory hidden as it is meant to be accessed by the developer as part of the migration process. Also, just calling it `migrate` may not be sufficiently clear that it is meant to be modified/edited/viewed as part of migration imo.